compute_endpoint/globus_compute_endpoint/endpoint/config/utils.py

```diff
@@ -165,6 +166,32 @@ def get_config(endpoint_dir: pathlib.Path) -> Config:
     return config
 
 
+def load_user_config_schema(endpoint_dir: pathlib.Path) -> dict | None:
+    from globus_compute_endpoint.endpoint.endpoint import Endpoint
+
+    user_config_schema_path = Endpoint.user_config_schema_path(endpoint_dir)
+    if not user_config_schema_path.exists():
+        return None
+
+    try:
+        return json.loads(user_config_schema_path.read_text())
+    except json.JSONDecodeError:
+        log.error("The user config schema is not valid JSON")
+        raise
```

Comment:
A the moment, I don't think this `log.error` call adds much value over the traceback.  At the point an admin see this error message, they'll likely have already seen the traceback below it.  Suggest one of:

1. not catching anything and letting the traceback share the information
2. sharing slightly _more_ information in the log (e.g., the path to the specific file, which will otherwise not be shared in the log), separating by newlines to draw attention to it (i.e., `log.error("\n\n [...] \n")`), perhaps including the exception text
3. Doing _just_ item 2 in the normal case, include the traceback as a debug measure (i.e., `log.debug(..., exc_info=True)`), but still exit with failure.

I offer similar suggestions for the `raise`s in the following function as well, so I won't make another comment thread about it.

Reply:
The logs add some clarity as to what is failing beyond what the traceback alone provides.

Per your suggestion in item 2, I've added newline characters and the schema file path to the relevant logs. I think the traceback is useful in each scenario, so I will leave it as-is.